### PR TITLE
changed area from extra to default return value

### PR DESCRIPTION
**Pull request by sjain28**
Resolving http://manage.dimagi.com/default.asp?236408

This fix makes it so that the return value of the area mapper intent is the measured area. 
Previously the area was an optional extra value, and there was no Intent extra called "odk_intent_data" as expected in the IntentCallout class.

### CODE REVIEW

File: app/src/main/java/richard/chard/lu/android/areamapper/activities/AreaMapperActivity.java [hunk 1]
```diff
@@ -450,10 +450,7 @@ public void run() {
             case R.id.button_ok:
 
                 Bundle result = new Bundle();
-                result.putString(
-                        EXTRA_KEY_AREA,
-                        Double.toString(areaCalculator.getArea())
-                );
+
                 result.putString(
                         EXTRA_KEY_PERIMETER,
```
Comment (amstone326) [inline app/src/main/java/richard/chard/lu/android/areamapper/activities/AreaMapperActivity.java:455]: Should we possibly be storing the perimeter in the same way as the area?

File: .idea/libraries/acra_4_6_1.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<component name="libraryTable">
```
Comment (amstone326) [inline .idea/libraries/acra_4_6_1.xml]: What are all of these new xml files for?

File: app/src/main/java/richard/chard/lu/android/areamapper/activities/AreaMapperActivity.java (hunk 1 above)
Comment (sjain28) [reply app/src/main/java/richard/chard/lu/android/areamapper/activities/AreaMapperActivity.java:455]: The callout needs to have 1 return value - either area or perimeter. The other can be returned as an optional extra. 

I assumed that area would be the default return value...but I could change that, or make it configurable.

Comment (amstone326) [reply app/src/main/java/richard/chard/lu/android/areamapper/activities/AreaMapperActivity.java:455]: Ah sorry ignore me, wasn't reading carefully. Definitely makes sense to add perimeter to the `EXTRA_KEY_RESPONSE_BUNDLE`

**Pull request merged**